lib/matplotlib/tests/test_colors.py

```diff
@@ -1981,7 +1981,7 @@ def test_mult_norm_call_types():
     mn.vmin = (-2, -2)
     mn.vmax = (2, 2)
 
-    vals = np.arange(6).reshape((3,2))
+    vals = np.arange(6, dtype='int64').reshape((3, 2))
```

Comment:
I assume what fails is the asserion below

```
assert no_norm_out[1].dtype == np.dtype('int64')
```

If so, wouldn't it be slightly better to leave `vals` as is and change the assert to
```
assert no_norm_out[1].dtype == vals.dtype
```
?

Reply:
Yes, fair, that would work as well.